LuaMenu/widgets/chobby/components/configuration.lua

```diff
@@ -89,7 +89,7 @@ function Configuration:init()
 
 	self.loadLocalWidgets = false
 	self.displayBots = false
-	self.displayBadEngines2 = true
```

Comment:
I want this to be true because wrapper launch can launch into other engine versions.

Reply:
I'm not sure I want to touch this because it was changed to fix an issue: https://github.com/Spring-Chobby/Chobby/commit/3627fe895d1fcffd08f244904afd5aa522f83c50
Was this change wrong?